typescript/src/optimistic-minting.ts

```diff
@@ -0,0 +1,52 @@
+import { BigNumber } from "ethers"
+import { TransactionHash } from "./bitcoin"
+import { Identifier, Event, TBTCVault } from "./chain"
+
+export type OptimisticMintingRequestedEvent = {
+  minter: Identifier
+  depositKey: BigNumber
+  depositor: Identifier
+  amount: BigNumber
+  fundingTxHash: TransactionHash
+  fundingOutputIndex: number
+} & Event
+
+export type OptimisticMintingRequest = {
+  requestedAt: number
+  finalizedAt: number
```

Comment:
Can we explain in a comment what `0` value means?

https://github.com/keep-network/tbtc-v2/blob/15be27edf4d852a284cf0a9ae5b9f422ef8d2cab/solidity/contracts/vault/TBTCOptimisticMinting.sol#L40-L42

Reply:
695ef2d30742d6a3910607fc54ea8f2c9a430203